plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -361,7 +363,32 @@ func ClusterRoles() []rbac.ClusterRole {
 				eventsRule(),
 			},
 		},
+		{
+			// a role making the csrapprover controller approve a node client CSR
+			ObjectMeta: metav1.ObjectMeta{Name: "system:csr-approver:nodeclient"},
```

Comment:
sharding by api group and possibly resource seems like a pattern that would work better in the future, e.g. `system:certificates.k8s.io:certificatesigningrequests:nodeclient`